content/boards/arduino-mega-2560-pro-mini/index.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: Arduino Mega 2560 Pro Mini
 description: Details on MobiFlight support for the Arduino Mega 2560 Pro Mini
+weight: 30
```

Comment:
Weight describes the sort order for the pages in the sidebar so they shouldn't all be the same. Unless you want to order in a way other than alphabetical you can just remove this. If you do want to sort in some other order give each page an increasing number in 10s (to allow for additional file insertion later, ah, BASIC days!)

Reply:
We discussed this in Discord, you pointed out that unweighted pages sink to the bottom so this change makes sense.